# Integrative Omics Data Analysis to Identify Shared Genes and Pathways Across Neurodegenerative Diseases


This research dissertation is submitted for the MSc in Bioinformaticsat Queen Mary, University of London.

Centre for Neuroscience, Surgery, and Trauma, Blizard Institute
Barts and the London School of Medicine and Dentistry

Supervisor: Dr Sarah Louise Morgan

## Description
Between 2015 and 2050 the proportion of the world’s population over 60 will
nearly double from 12% to 22%. With an aging population comes the increased risk of
neurodegenerative diseases, a group of disorders effecting neurons in the brain resulting in
their progressive atrophy. Despite years of research, a cure is yet to be found, highlighting the
importance of continued research of therapeutic options. This can only be achieved by
understanding the implicated genes and pathways that can be targeted with treatment. Many
neurodegenerative diseases share similarities in risk factors, genetic determinants, symptoms,
pathology and disease mechanisms, prompting the hypothesis that these diseases are different
manifestations of the same underlying disease and treatment for one may be able to treat
others.

This study aimed to identify overlapping dysfunctional genes and pathways across the
following diseases to determine potential drug targets: Alzheimer’s disease, Parkinson’s
disease, amyotrophic lateral sclerosis, Lewy body dementia, Huntington’s disease and
multiple sclerosis.

## Data
Data was gathered from the most comprehensive genome-wide association study for each disease and filtered to retain only significant genetic variants. Data for Lewy body dementia and amyotrophic lateral sclerosis was obtained from the GWAS catalog: https://www.ebi.ac.uk/gwas/. Data for the other studies were requested directly from the research team of the orignal studies.

Data requested directly from the researchers cannot be uploaded or shared due to data protection, whilst files for the publicly available data also cannot be uploaded on github due to its large file size.

Information on the orignal GWAS data used for each disease can be found in the supplementary_materials file.

## Scripts
Scripts were run in RStudio version (2023.06.0+421)

## Contact 
Khadija Paderwala- bt22050@qmul.ac.uk
